<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digital Oscilloscope</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="gCheck">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>81</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Grid</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>t = 1s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>t = 100ms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>t = 10ms</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="aBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A = 1V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A = 100mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A = 10mV</string>
     </property>
    </item>
   </widget>
   <widget class="Canvas" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>79</y>
      <width>671</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>0</y>
      <width>671</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlayout">
     <item>
      <widget class="QWidget" name="navigation" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
